core/server/config/overrides.json

```diff
@@ -6,6 +6,7 @@
         "helperTemplates": "core/server/helpers/tpl/",
         "adminViews": "core/server/web/admin/views/",
         "defaultViews": "core/server/views/",
+        "defaultSettings": "core/settings/",
```

Comment:
> Inside of Ghost, there will be a copy of the default version of these files, e.g. in core/settings or core/services/settings

Why `core/settings`? 🤔 This would mean these are files shared by client and server.

I think @ErisDS meant `core/server/settings`?

Also: i am not sure we should create a new root folder in the server root dir. I don't see the real advantage yet.
My gut feeling tells me to keep the default files in the service settings folder.

Reply:
> I think @ErisDS meant core/server/settings?

Maybe, but that's what I took from the original spec:

> We also keep track of the default template inside of Ghost core e.g. in core/settings

---

> My gut feeling tells me to keep the default files in the service settings folder.

Agreed, or `core/server/settings`, just like we have the `core/server/views`, maybe?

Reply:
> core/server/views

I am not sure `views` makes sense at the root level 🤔 . Because they actually only belong to the `site` app.
We register them as default views and you are then be able to render e.g. `error`.
We also have `web/admin/views`.

Reply:
I think we should use `core/server/settings` for now and maybe have decision later about this.

Reply:
@kirrg001 I assume the currently chosen destination for the default settings files in `core/server/settings` is ok now? Still had this as my todo:

>  decide on final location for default yaml files

Reply:
> My gut feeling tells me to keep the default files in the service settings folder.

> There should absolutely only be 1 folder in core that holds all of the logic to do with the new settings behaviour.

I would put the default routes file into this location for now. I still don't see any advantage in creating a new root folder.

> Also: i am not sure we should create a new root folder in the server root dir. I don't see the real advantage yet.

Reply:
Alright. I changed this back. The default file is now in `core/services/settings` and there is no additional root folder.